plugins/modules/wafv2_web_acl.py

```diff
@@ -267,22 +334,27 @@ def __init__(self, wafv2, name, scope, fail_json_aws):
         self.fail_json_aws = fail_json_aws
         self.existing_acl, self.id, self.locktoken = self.get_web_acl()
 
-    def update(self, default_action, description, rules, sampled_requests, cloudwatch_metrics, metric_name):
+    def update(self, default_action, description, rules, sampled_requests, cloudwatch_metrics, metric_name, custom_response_bodies):
+        req_obj = {
+            'Name': self.name,
+            'Scope': self.scope,
+            'Id': self.id,
+            'DefaultAction': default_action,
+            'Description': description,
+            'Rules': rules,
+            'VisibilityConfig': {
+                'SampledRequestsEnabled': sampled_requests,
+                'CloudWatchMetricsEnabled': cloudwatch_metrics,
+                'MetricName': metric_name
+            },
+            'LockToken': self.locktoken
+        }
+
+        if len(custom_response_bodies) > 0:
```

Comment:
```suggestion
        if custom_response_bodies:
```

requires the changes below (remove default value)